.github/workflows/build.yml

```diff
@@ -23,5 +24,19 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install tox tox-gh-actions
+      - name: Install System Dependencies (macOS)
+        if: runner.os == 'macOS'
+        run: |
+          brew install pkg-config
+          brew install libffi
+          brew install pango
+          brew install glib
+      - name: Install System Dependencies (Windows)
+        if: runner.os == 'windows'
+        run: |
+          curl -L https://github.com/naveen521kk/pango-build/releases/download/v1.48.0/pango-build-win64.zip -o pango.zip pango.zip
```

Comment:
<img width="998" alt="Screenshot 2020-12-13 at 09 40 30" src="https://user-images.githubusercontent.com/3501061/102008335-82a80380-3d27-11eb-9da7-c40983f404c7.png">

I notice the warning "Could not resolve host pango.zip" shows up in the logs. I believe removing the last argument fixes this issue.

```suggestion
          curl -L https://github.com/naveen521kk/pango-build/releases/download/v1.48.0/pango-build-win64.zip -o pango.zip
```

Reply:
Oh! Yeah a typo.